storybookjs/storybook
# fix(webpack-dev-server) : enable webpack to rebuild changes in node_modules by default

Issue: https://github.com/storybooks/storybook/issues/5572

## What I did

I removed the "ignore: /node_modules/" from watchOptions.
Webpack should be able to rebuild changes inside node_modules by default
Even on very huge projects, I never had any performance issue, so I don't see the point of ignoring the node_modules.

## How to test

You have to run storybook, and changea file inside the `node_modules` folder.
Webpack should be able to see the change, and rebuild accordingly.

- Is this testable with Jest or Chromatic screenshots?
I don't think so
- Does this need a new example in the kitchen sink apps?
No idea
- Does this need an update to the documentation?
No

If your answer is yes to any of these, please make sure to include it in your PR.
